### ISSUE TITLE
Please allow using gzipped log files

### ISSUE THREAD

**Issue by jech** [allinurl/goaccess#2141]
Under many Linux distributions, logfiles are compresses after a few days.  It would be convenient if goacess could transparently decompress such compressed log files.

**Comment by dertuxmalwieder** [comment]
Possible ways to fix this:

1. Actually manage your server so it does *not* do things you don't want it to do.
2. Pipe the output of `zcat` instead of the actual log files to goaccess.

**Comment by jech** [comment]
Yes, I know about the workaround.  I'm just saying that it would be convenient if goaccess could decompress on the fly.

**Comment by 0bi-w6n-K3nobi** [comment]
Hi @jech 

Perhaps you can a fuse filesystem that have to support that.
I find fuse-bgzip [here](https://github.com/sahlberg/fuse-bgzip).

**Comment by allinurl** [comment]
@jech Thanks for suggesting this. I can look into this. I'm guessing it will require to build `--with-zlib`, it shouldn't be too bad. @0bi-w6n-K3nobi Thanks for sharing those workarounds, certainly did not know the fuse-bgzip :)
